### PR TITLE
Test activation of external PR on travis

### PR DESCRIPTION
**Pull request by benzonico**

### CODE REVIEW

File: sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java [hunk 1]
```diff
@@ -77,6 +77,7 @@ public void describe(SensorDescriptor descriptor) {
 
   @Override
   public void execute(SensorContext context) {
+    System.out.println("EXECUTE DA SENSOR");
```
Comment (sonartech-externalpr) [inline sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

**Comment by sonartech-externalpr** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java [hunk 2]
```diff
@@ -77,6 +77,8 @@ public void describe(SensorDescriptor descriptor) {
...
+    System.out.println("EXECUTE DA SENSOR, FOR REAL");
```
Comment (sonartech-externalpr) [inline sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

**Comment by sonartech-externalpr** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by benzonico** [comment]
This was a test of external PR activation.

**Pull request closed without merging**